eppoclient/applicationlogger.go

```diff
@@ -1,14 +1,64 @@
-package applicationlogger
+package eppoclient
```

Comment:
removed the `applicationlogger` package and inlined into this file - simpler to just keep everything in one package.

Reply:
❗ This is a breaking change and requires a major version bump. I would rather not do this as it doesn't seem required to fix the profiler